app/views/pieces/show.html.erb

```diff
@@ -0,0 +1,52 @@
+<h1>Pieces show page goes here</h1>
+
+<!-- <script>
+
+$(function() {
+  $('.square a').click(
+    function submitPieceMove(id, file, rank) {
+      // console.log(id);         // Error, the id is not passed, results in 
+                                  // wrong routing /pieces/[object%20Object]
+      $.ajax("/pieces/" + id, {
+        method: "PATCH",
+        data: {
+          file: file,
+          rank: rank
+        }
+      });
+  });
+});
+
+</script> -->
+
+<div class="chessboard">
+
+  <div class="pieces">
+    <% file = 0 %>
+    <% rank = 9 %>
+
+    <% 8.times do %>
+      <% rank = rank - 1 %>
+      <% 8.times do %>
+        <% file = file + 1 %>
+          <div class="square" id="<%= [file, rank] %>">
+            <% if @piece_coord == [file, rank] %>
+              <div class="highlight">
+                <p><%= "#{file}, #{rank}" %></p>
+              </div>
+            <% else %>
+              <div>

```

Comment:
Adding a css class with some padding would help out with your move display.  Currently, all your buttons are scrunched together making it difficult to accurately see the board.  See the comment in pieces.scss for recommendatios on what to add